RecoHI/HiMuonAlgos/python/HiRegitMuonDetachedTripletStep_cff.py

```diff
@@ -64,29 +64,31 @@
 
 # fitting: feed new-names
 hiRegitMuDetachedTripletStepTracks                 = RecoTracker.IterativeTracking.DetachedTripletStep_cff.detachedTripletStepTracks.clone(
-    AlgorithmName = cms.string('undefAlgorithm'),
+    AlgorithmName = cms.string('hiRegitMuDetachedTripletStep'),
     src                 = 'hiRegitMuDetachedTripletStepTrackCandidates'
 )
 
 
 import RecoTracker.FinalTrackSelectors.multiTrackSelector_cfi
+import RecoHI.HiTracking.hiMultiTrackSelector_cfi
 hiRegitMuDetachedTripletStepSelector               = RecoTracker.FinalTrackSelectors.multiTrackSelector_cfi.multiTrackSelector.clone(
```

Comment:
Selectors have to be cloned from RecoHI.HiTracking.hiMultiTrackSelector_cfi.hiMultiTrackSelector object. PR will be updated.